packages/manager/apps/web-ongoing-operations/src/components/OngoingOperationDatagridBadge/OngoingOperationDatagridBadge.tsx

```diff
@@ -39,47 +34,40 @@ export default function OngoingOperationDatagridBadge({
         return ODS_BADGE_COLOR.information;
     }
   };
+
+  const printIconAndTooltip: boolean =
+    props.status === 'todo' ||
```

Comment:
can you put these status into a data structure please ? either an enum or const enum